enterprise/releases/2024-10-0.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: '2024.10.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2024.10.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+- **Important:** If you were previously running a version older than v2024.04.X we highly recommend first upgrading to v2024.04.5 and staying there for 7 to 35 days
+  - The v2024.04.0 release introduced a backend change for how query statistics are stored. Starting that release the old format was still supported in a dual write mode.
+  - Starting with this release the new packed format is required and the only one supported.
+  - If you have not been on v2024.04.X for sufficient time the migrations will fail with an error to prevent accidental loss of query statistics.
+- This release removes the use of the **Object storage**
+  - With the new packed storage format, pganalyze Enterprise Server now stores all data within the main database, including log texts
+  - This potentially affects database storage size, although the pganalyze team has actually observed storage reductions due to the packed format on our cloud-based environment, despite the extra data being stored
+  - Additionally we recommend reviewing your encryption configuration for your database server, since pganalyze now relies on the database storage to be encrypted for security of log texts (instead of a separate log encryption key)
+  - After upgrading the installation will still be able to read from the object storage for old logs, but no longer write to it. After 7 days have passed (the log retention time) you can remove all object storage related configuration.
+
+## Features
+
+- Index Advisor
+  - Add support for "pg_bigm", "cube" and "earthdistance" extensions <!-- 3953 -->
```

Comment:
Based on the broken syntax highlighting, GitHub does not seem to like these HTML comments in markdown like this (so other markdown tools may also). But I guess if it doesn't seem to cause any problems, that's fine.

Reply:
Yeah, I've done this in previous releases, I think its fine despite the bad syntax highlighting.